### PR TITLE
Pr comments

### LINKED ISSUE #213: Code creates directories with colons which aren't supported in Windows.
This happens in a few places.  Consider src/routines/write_models.jl write_model_results().  The timestamp will have colons.

```
new_folder = mkdir("$save_path/$(round(Dates.now(),Dates.Minute))")
```

Windows has paths like c:\data, so this will crash at runtime.

### PR DESCRIPTION
**Pull request by jd-lara**
fixes #213 and #217

### CODE REVIEW

File: src/devices/device_models/common/model_utils.jl [hunk 1]
```diff
@@ -31,3 +31,7 @@ function _remove_underscore(original::Symbol)
 
     return parts[1]
 end
+
+function remove_char(s::String, char::String)
+    return replace(s, Regex("[$char]") => "")
```
Comment (daniel-thom) [inline src/devices/device_models/common/model_utils.jl]: What do you think about replacing ":" with "-" or "_"?

File: src/routines/make_initial_conditions.jl [hunk 2]
```diff
@@ -98,8 +99,9 @@ function duration_init(canonical_model::CanonicalModel,
             ini_conds = canonical_model.initial_conditions[key] = Vector{InitialCondition}(undef, length_devices)
 
             for (ix, g) in enumerate(devices)
-                times = [999.0*(PSY.get_activepower(g) > 0),  # Time on
-                         999.0*(PSY.get_activepower(g) <= 0)] # Time off
+                time_on = (PSY.get_activepower(g) > 0) ? 999.0 : 0.0
```
Comment (daniel-thom) [inline src/routines/make_initial_conditions.jl]: Magic number.  Can 999.0 be defined somewhere?

File: src/devices/device_models/common/model_utils.jl (hunk 1 above)
Comment (daniel-thom) [reply src/devices/device_models/common/model_utils.jl]: Also, I think it would be better to use the normal, non-Regex version of the function.

Comment (jd-lara) [reply src/devices/device_models/common/model_utils.jl]: The purpose is to have a function that can do several character removals and replacements. See 05912f8

File: src/routines/make_initial_conditions.jl [hunk 3]
```diff
@@ -1,3 +1,5 @@
+const missing_initial_conditions_time_count = 999.0
```
Comment (daniel-thom) [inline src/routines/make_initial_conditions.jl]: constants should be in the form of MISSING_INITIAL_CONDITIONS_TIME_COUNT

File: src/devices/device_models/common/model_utils.jl (hunk 1 above)
Comment (daniel-thom) [reply src/devices/device_models/common/model_utils.jl]: A docstring would make that clear. :)  Also, maybe the function should be plural.

**Pull request merged**